src/stylesheets/components/_accordions.scss

```diff
@@ -81,6 +82,31 @@ $accordion-border: 3px solid $color-gray-lightest;
     > li {
       @include accordion-list-item-styles;
     }
+  }
+
+  [aria-expanded=false] {
```

Comment:
These additions might be a merge conflict problem as this code should be in the mixins in the above file.

Did you want to make any changes to the accordion style? You might want to get rid of these changes and make your changes manually if it's hard to figure out the conflicts. I may have gotten something wrong when I was working on it with you.